src/pages/CreateProposal/index.tsx

```diff
@@ -241,7 +241,7 @@ ${bodyValue}
   return (
     <Trace page={PageName.VOTE_PAGE} shouldLogImpression>
       <PageWrapper>
-        <AppBody {...{ maxWidth: '800px' }}>
+        <BodyWrapper chainId={chainId} maxWidth="800px">
```

Comment:
feels like BodyWrapper should handle getting the chainId itself vs passing around so many levels deep. I also am not clear why BodyWrapper is adding the glow vs doing it on the component itself.